src/app/components/progress-indicator/index.html

```diff
@@ -19,26 +19,26 @@
       isOptional="true"
       defaultValue="Vertical"
     >
-      Specifies whether to display the progress indicator vertically or horizontally. For <a routerLink="/components/wizard">modal wizards</a>, use the horizontal display mode. For waterfall pages and passive progress indicators, use the vertical display mode.
+      Specifies whether to display the progress indicator vertically or horizontally. For <a routerLink="/components/wizard">modal wizards</a>, use the horizontal display mode. For waterfall pages and passive progress indicators, use the vertical display mode. This property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="isPassive"
       isOptional="true"
       defaultValue="false"
     >
-      Indicates whether the progress indicator is passive. Passive progress indicators inform users of progress that concerns them but that they are not responsible for, and they must use the vertical display mode.
+      Indicates whether the progress indicator is passive. Passive progress indicators inform users of progress that concerns them but that they are not responsible for, and they must use the vertical display mode. This property accepts <stache-code>boolean</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="messageStream"
       isOptional="true"
     >
-      Specifies an observable of <stache-code>SkyProgressIndicatorMessageType</stache-code> that determines the status to display for progress items in the progress indicator. The message stream is a queue of commanding messages to change the state of the progress indicator based on one of the three message types.
+      Specifies an observable of <stache-code>SkyProgressIndicatorMessageType</stache-code> that determines the status to display for progress items in the progress indicator. The message stream is a queue of commanding messages to change the state of the progress indicator based on one of the three message types. This property accepts <stache-code>string</stache-code> values.
```

Comment:
I'd link from this to the SkyProgressIndicatorMessageType properties section. I'd make "an observable of SkyProgressIndicatorMessageType" and/or "one of the three message types" the link text. In an `a` tag, set `stacheRouterLink` equal to "." and set `fragment` to "skyprogressindicatormessagetype-properties" ... If you haven't seen a link like this in the SKY UX docs, you can look at the checkbox component docs for examples because it has internal links to the demo and the accessibility section.

Reply:
I went with the first option you presented.

Reply:
Looks good.